it/src/main/java/org/corfudb/universe/cluster/docker/DockerCluster.java

```diff
@@ -0,0 +1,197 @@
+package org.corfudb.universe.cluster.docker;
+
+import com.google.common.collect.ImmutableMap;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.messages.NetworkConfig;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.universe.cluster.Cluster;
+import org.corfudb.universe.cluster.ClusterException;
+import org.corfudb.universe.service.DockerService;
+import org.corfudb.universe.service.Service;
+import org.corfudb.universe.service.Service.ServiceParams;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Function;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.function.Function'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)